Bottom: 76cadb7aea600da6bd4f17b687c39630d0292538
Top:    d4fa4db4846829c1cf64e094e4d9a042c8034616
Author: Jason Wang <jasowang@redhat.com>
Date:   2015-10-12 13:23:35 +0800

Refresh of vl-c-init-delayed-object-after

---

diff --git a/vl.c b/vl.c
index 366f38f..17ae26c 100644
--- a/vl.c
+++ b/vl.c
@@ -2774,7 +2774,7 @@ static bool object_create_initial(const char *type)
 
 /*
  * The remainder of object creation happens after the
- * creation of chardev, fsdev and device data types.
+ * creation of chardev, fsdev, net clients and device data types.
  */
 static bool object_create_delayed(const char *type)
 {
